packages/bff/src/services/catalogService.ts

```diff
@@ -122,5 +123,52 @@ export function catalogServiceBuilder(
 
       return response;
     },
+    createDescriptor: async (
+      eServiceId: string,
+      eServiceDescriptorSeed: bffApi.EServiceDescriptorSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<CreatedResource> => {
+      const { id } = await catalogProcessClient.createDescriptor(
```

Comment:
Return directly the function or the response, without destructuring

Reply:
I thought we only needed to pass the ID, but then I realized there is a passthrough.

Reply:
Actually from the OpenApi these routes should return only the ID. Am I missing something?